wasm/index.mjs

```diff
@@ -51,3 +51,12 @@ async function load(module, imports) {
     }
   }
 }
+
+async function fetchOrReadFromFs(inputPath) {
+  try {
+    const fs = await import('fs');
+    return fs.readFileSync(inputPath);
```

Comment:
Would we need to use a different API for Deno? Or can Deno `fetch` a `file://` url?

Reply:
I do know that deno can instantiate streaming ([full docs](https://deno.land/manual@v1.12.2/getting_started/webassembly)), but I'm not sure about the consequences of deno's design are.

Reply:
well, i spun up a deno deploy playground and tried to `await import('fs')` and got this error (as expected)

```
TypeError: Relative import path "fs" not prefixed with / or ./ or ../ from "file:///src/main.ts"
```

after that, i wondered if it currently even works in deno so i tried to run the snippet from the [docs](https://lightningcss.dev/docs.html#from-deno-or-in-browser) and got a similar error:

```
Relative import path "napi-wasm" not prefixed with / or ./ or ../ from
```

i then got it to work after i added an [import map](https://deno.land/manual@v1.29.4/basics/modules/import_maps):
```json
{
  "imports": {
    "napi-wasm": "https://unpkg.com/napi-wasm"
  }
}
```

then i tested passing a full url to `init` and got it work

```js
import { serve } from 'https://deno.land/std@0.155.0/http/server.ts';
import init, { transform } from 'https://unpkg.com/lightningcss-wasm';

await init('https://unpkg.com/lightningcss-wasm/lightningcss_node.wasm');

let {code, map} = transform({
  filename: 'style.css',
  code: new TextEncoder().encode('.foo { color: red }'),
  minify: true,
});

const css = new TextDecoder().decode(code);

serve((req: Request) => new Response(css)); // .foo{color:red}
```

all of that to say, my current PR should be fine as long as the import map is added.